feelpp/feel/feelfilters/hbf.cpp

```diff
@@ -543,7 +542,6 @@ template void writeHBF( std::string const& , holo3_image<int8_t> const& );
 template void writeHBF( std::string const& , holo3_image<int16_t> const& );
 template void writeHBF( std::string const& , holo3_image<int32_t> const& );
 template void writeHBF( std::string const& , holo3_image<int64_t> const& );
-template void writeHBF( std::string const& , holo3_image<long> const& );
```

Comment:
long type is the same as int64_t or int32_t depending on the compiler. In both cases, the specialization for long type was useless and was causing errors